pkgs/development/python-modules/pybluez/default.nix

```diff
@@ -8,7 +8,7 @@
 
 buildPythonPackage rec {
   pname = "pybluez";
-  version = "unstable-2022-01-28";
+  version = "unstable-0.23";
```

Comment:
?

Reply:
Nix-update gone wrong, dropped the commit.